lib/utils/read-user-info.js

```diff
@@ -9,14 +9,16 @@ exports.password = readPassword
 exports.username = readUsername
 exports.email = readEmail
 
-function read (opts) {
-  return Promise.resolve().then(() => {
-    log.clearProgress()
-    return readAsync(opts)
-  }).then(() => log.showProgress(), er => {
+async function read (opts) {
+  log.clearProgress()
+  try {
+    const input = await readAsync(opts)
+    log.showProgress()
+    return input
+  } catch (er) {
     log.showProgress()
     throw er
-  })
+  }
```

Comment:
another alternative for the entire function body, since node 10+ is required:
```suggestion
  log.clearProgress()
  return readAsync(opts).finally(() => log.showProgress())
```

Reply:
I like this approach. Thank you Jordan!